src/main/java/com/google/enterprise/cloudsearch/fs/FileDelegate.java

```diff
@@ -198,6 +199,24 @@
    */
   void startMonitorPath(Path watchPath, RepositoryEventPusher eventPusher) throws IOException;
```

Comment:
Where is this original overload still used?

Reply:
In tests.